pkg/build/nodeimage/const_storage.go

```diff
@@ -20,43 +20,49 @@ package nodeimage
 The default PV driver manifest and images are provisionally rancher.io/local-path-provisioner
 NOTE: we have customized it in the following ways:
 - storage is under /var instead of /opt
-- debian-base is used as the helper image (k8s already ships this upstream as the base for many images) instead of busybox
-- schedule to linux nodes
+- our own image and helper image
+- schedule to linux nodes only
 - install as the default storage class
 */
 
-var defaultStorageImages = []string{"docker.io/rancher/local-path-provisioner:v0.0.14", "k8s.gcr.io/build-image/debian-base:buster-v1.7.2"}
+const storageProvisionerImage = "docker.io/kindest/local-path-provisioner:v0.0.22-kind.0"
+const storageHelperImage = "docker.io/kindest/local-path-helper:v20220512-507ff70b"
+
+// image we need to preload
+var defaultStorageImages = []string{storageProvisionerImage, storageHelperImage}
 
 const defaultStorageManifest = `
-# kind customized https://github.com/rancher/local-path-provisioner manifest
 apiVersion: v1
 kind: Namespace
 metadata:
   name: local-path-storage
+
 ---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: local-path-provisioner-service-account
   namespace: local-path-storage
+
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: local-path-provisioner-role
 rules:
-- apiGroups: [""]
-  resources: ["nodes", "persistentvolumeclaims"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: [""]
-  resources: ["endpoints", "persistentvolumes", "pods"]
-  verbs: ["*"]
-- apiGroups: [""]
-  resources: ["events"]
-  verbs: ["create", "patch"]
-- apiGroups: ["storage.k8s.io"]
-  resources: ["storageclasses"]
-  verbs: ["get", "list", "watch"]
+  - apiGroups: [ "" ]
+    resources: [ "nodes", "persistentvolumeclaims", "configmaps" ]
+    verbs: [ "get", "list", "watch" ]
+  - apiGroups: [ "" ]
+    resources: [ "endpoints", "persistentvolumes", "pods" ]
```

Comment:
are endpoints used for anything that is not leader election ?
because it seems is disabled now

Reply:
I didn't look into this, this portion is straight from the upstream deployment unmodified.
The indent changed upstream, I rebuilt this string by starting again from the unmodified latest upstream standard deployment yaml and then applying our changes (listed above)